<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>442</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="logo">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>logoProjet_petit.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>22</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Bienvenue sur JVT Hotels !</string>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Veuillez vous connecter </string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="topMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_signup">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Je n'ai pas de compte ...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_val">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Valider</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_radbtn">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QRadioButton" name="radioAdmin">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Administrateur</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QRadioButton" name="radioClient">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Client</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QRadioButton" name="radioEmploye">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Employ√©</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_mdp">
         <property name="spacing">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="label_mdp">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>Mot de passe</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_mdp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_mail">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="label_id">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>Mail</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_id">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_10"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_mdp</tabstop>
  <tabstop>radioAdmin</tabstop>
  <tabstop>radioClient</tabstop>
  <tabstop>radioEmploye</tabstop>
  <tabstop>pushButton_signup</tabstop>
  <tabstop>pushButton_val</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
